index.html

```diff
@@ -0,0 +1,86 @@
+                <li><a href="" class="close-menu"><i class="fas fa-times"></i></a></li>
+            </ul>
+        </nav>
+        <div class="home-container">
+            <img src="images/home-bg16.png" alt="">
+            <div class="heading">
+                <h3>"Hello! Poetry Lovers"</h3>
+                <h1>ETHIOPIA POETIC JAZZ 2013 EC</h1>
+                <p class="intro">Beloved families of Tobia, our last stage of the year will
+                   be held on August 14 at Intercontinental Hall starting from 11.30 you are invited.
+                </p>
+                <p class="time-date">2013.12.10 (Ethiopian Calendar)</p>
+                <p class="venue">@Ethiopia Hotel, Art Center</p>
+            </div>
+            <div class="main-program">
+                <h2>Main Program</h2>
+                <hr>
+                <div class="program-container">
+                    <div class="programs">
+                        <i class="fas fa-feather icon"></i>
+                        <p class="program-title">Poetry</p>
+                        <p class="program-text">Country’s famous and professional poets as well as new up coming ones will have audience moved with some of their best work.</p>
+                    </div>
+                    <div class="programs">
+                        <i class="fas fa-music icon"></i>
+                        <p class="program-title">Live Music</p>
+                        <p class="program-text">Talented traditional jazz musicians will be filling the air in between performances to keep the audience on their feet!</p>
+                    </div>
+                    <div class="programs">
+                        <i class="fas fa-theater-masks icon"></i>
+                        <p class="program-title">Theatric Act</p>
+                        <p class="program-text">Breathtaking live theatric shows will be an important part of the show.</p>
+                    </div>
+                    <div class="programs">
+                        <i class="fas fa-chalkboard-teacher icon"></i>
+                        <p class="program-title">Motivational Speakers</p>
+                        <p class="program-text">Informative and inspirational speeches by some of the best talented and self-aware motivational speakers.</p>
+                    </div>
+                </div>  
+                <button class="btn">Join Poetic Jazz Event 2013</button>
+                <a class="desktop-btn" href="#">SEE THE WHOLE PROGRAM</a>
+            </div>
+            <div class="partner partner-desktop-home">
+                <h2>Partner</h2>
+                <hr>
+                <img src="images/et-airlines.png" alt="ethiopian airlines logo">
+                <img src="images/negus-logo.png" alt="Negus logo">
+                <img src="images/raya-coffee.png" alt="Raya coffee logo">
+            </div>
+            <footer class="footer desktop-home-footer ">
```

Comment:
- [ ] The _partner_ and _footer_ sections are missing in your design in your mobile version. Kindly ensure to improve upon these aspects by adding the _partner_ and _footer_ sections to your mobile version of your design.

**Your mobile bottom part:**
![image](https://user-images.githubusercontent.com/12745474/132688774-169eebc2-3822-411e-a648-b784d8bada5d.png)


**The original mobile bottom part:**
![image](https://user-images.githubusercontent.com/12745474/132610415-16677bde-c76a-43a9-a103-d2fc9dbe06f8.png)

Reply:
Hello 
Can you be a little clear about this? I understood it the other way. The partners and footer sections are present on the about page, not the home page.